File: typescript-client-sdk/src/sdk/models/operations/getallcrmintegrations.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetAllCrmIntegrationsResponseBody extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=integrations", elemType: shared.Integration })
  integrations?: shared.Integration[];
}


export class GetAllCrmIntegrationsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  responseBody?: GetAllCrmIntegrationsResponseBody;
}

File: python-client-sdk/src/orbapi/models/operations/get_all_crm_integrations.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from orbapi import utils
from ..shared import integration as shared_integration


@dataclass_json
@dataclasses.dataclass
class GetAllCrmIntegrationsResponseBody:
    integrations: Optional[list[shared_integration.Integration]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('integrations') }})
    

@dataclasses.dataclass
class GetAllCrmIntegrationsResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    response_body: Optional[GetAllCrmIntegrationsResponseBody] = dataclasses.field(default=None)
    
